apps/coscrad-frontend/src/components/resources/shared/with-commands.tsx

```diff
@@ -1,16 +1,20 @@
-import { ICommandFormAndLabels } from '@coscrad/api-interfaces';
+import { AggregateCompositeIdentifier, ICommandFormAndLabels } from '@coscrad/api-interfaces';
 import { FunctionalComponent } from '../../../utils/types/functional-component';
 import { CommandPanel } from '../../commands';
 
 export const WithCommands =
     <TProps,>(
         WrappedComponent: FunctionalComponent<TProps>,
-        mapPropsToActions: (props: TProps) => ICommandFormAndLabels[]
+        mapPropsToActions: (props: TProps) => ICommandFormAndLabels[],
+        mapPropsToCommandContext: (props: TProps) => AggregateCompositeIdentifier
     ) =>
     (props: TProps) =>
         (
             <div>
                 {WrappedComponent(props)}
-                <CommandPanel actions={mapPropsToActions(props)} />
+                <CommandPanel
+                    actions={mapPropsToActions(props)}
+                    commandContext={mapPropsToCommandContext(props)}
```

Comment:
can this just simply be "command context"?

Reply:
Let’s wait to fix it as we might end up removing it altogether. We’ll see what happens on the ID generation story.